tests/sentry/tasks/test_daily_summary.py

```diff
@@ -502,13 +513,60 @@ def test_slack_notification_contents(self):
         # check performance issues
         assert "*Today's Top 3 Performance Issues*" in blocks[6]["text"]["text"]
         assert link_text.format(self.perf_event.group.id) in blocks[6]["text"]["text"]
+        assert "db - SELECT `books_author`.`id`, `books_author`.`..." in blocks[6]["text"]["text"]
         assert link_text.format(self.perf_event2.group.id) in blocks[6]["text"]["text"]
         # repeat above for second project
         assert self.project2.slug in blocks[8]["text"]["text"]
-        assert "*Today's Top 3 Error Issues" in blocks[9]["fields"][0]["text"]
-        assert link_text.format(self.group4.id) in blocks[9]["fields"][0]["text"]
+        assert "*Today’s Event Count*" in blocks[3]["fields"][0]["text"]
```

Comment:
this seems awfully brittle but i don't have any better solution :( 

Maybe we can just assert that the text exists, rathert than asserting it in the specific block? Or at least pull it out into a variable or something?
```
const text_fields = document.getElementById(...) (or however we're grabbing the element)
```

Reply:
yeah unfortunately it's kinda hard to do, all the block kit tests are like this. I could put each block into a variable in a separate PR